plugin/mouse/mouse.py

```diff
@@ -53,6 +55,12 @@
     default=0,
     desc="Pop noise clicks left mouse button. 0 = off, 1 = on with eyetracker but not with zoom mouse mode, 2 = on but not with zoom mouse mode",
 )
+mod.setting(
+    "mouse_enable_pop_wake",
+    type=int,
```

Comment:
Let's use a boolean as those are now supported

Reply:
changed type to bool